### PR TITLE
Very basic CT setup + tests for couch_replicator_scheduler

### PR DESCRIPTION
**Pull request by robfraz**
BugzID: 63012

### CODE REVIEW

File: test/couch_replicator_SUITE.erl [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+init_per_suite(Config) ->
+    DbDirs = setup_db_directories(?config(priv_dir, Config)),
+    lists:append(Config, DbDirs).
+
+init_per_testcase(_Name, Config) ->
+    [DbEtcDir, DbDataDir, DbLogDir] = 
+        [proplists:get_value(Key, Config) || Key <- [db_etc_dir, db_data_dir, db_log_dir]],
+    {ini_files, DefaultIniPath, LocalIniPath} = 
+        write_db_ini_files(DbEtcDir, DbDataDir, DbLogDir),
+
```
Comment (robfraz) [inline test/couch_replicator_SUITE.erl:26]: will remove this totally unnecessary whitespace

File: test/couch_replicator_SUITE.erl [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    application:set_env(config, ini_files, [DefaultIniPath, LocalIniPath]),
+
```
Comment (robfraz) [inline test/couch_replicator_SUITE.erl:28]: And this one.

**Comment by rnewson** [comment]
+1
